sycl/include/CL/sycl/atomic_ref.hpp

```diff
@@ -0,0 +1,662 @@
+//==----- atomic_ref.hpp - SYCL 2020 atomic_ref ----------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include <CL/__spirv/spirv_ops.hpp>
```

Comment:
Do we really need this include? It seems like we are using all SPIR-V functions through `detail/spirv.hpp` helper

Reply:
The header is included by `multi_ptr.hpp` too, so I've removed the include directive.